js/models/AppearanceSettings.js

```diff
@@ -0,0 +1,10 @@
+import BaseModel from './BaseModel';
+
+export default class extends BaseModel {
+  defaults() {
+    return {
+      showAdvancedVisualEffects : true,
+      windowControlStyle : 'mac'
```

Comment:
@jjeffryes ok, I see you touched on a lot of my points above in some comments below.